### PR TITLE
Ch 440

### PR DESCRIPTION
**Pull request by daniel-hlavaty**

### CODE REVIEW

File: src/components/ticketList/TicketListItem.tsx [hunk 1]
```diff
@@ -169,7 +164,7 @@ const TicketListItem: FC<ITicketListItem> = ({ index, ticket, loading, type, act
 								<Col md={12} span={24}>
 									<Button
 										disabledPopoverText={t('Matches are no longer open to copy')}
-										disabled={activeMatches?.length === 0} // If ticket with active matches is empty disable button
+										// disabled={activeMatches?.length === 0} // If ticket with active matches is empty disable button
```
Comment (michalbella) [inline src/components/ticketList/TicketListItem.tsx:167]: why is this commented?

**Review by michalbella — APPROVED** [review]

File: src/components/ticketList/TicketItem.tsx [hunk 2]
```diff
@@ -58,7 +58,9 @@ const TicketItem = ({ match, oddsInfo }: Props) => {
 
 	const isPlayedNow = () => {
 		if (oddsDataFromContract) {
-			if (match.market.isOpen && getMatchOddsContract(match.side, oddsDataFromContract) === '0' && !match.market.isPaused) return true
```
[the remark this answers is not in the document]
Comment (daniel-hlavaty) [reply src/components/ticketList/TicketItem.tsx:61]: We do, if match is currently beign played, flag is open remains true, but oddsDataFrom contract return 0.

File: src/components/ticketList/TicketList.tsx [hunk 3]
```diff
@@ -61,6 +63,8 @@ const TicketList: FC<ITicketList> = ({ type = TICKET_TYPE.OPEN_TICKET, list = []
 	const [copyModal, setCopyModal] = useState<{ visible: boolean; onlyCopy: boolean }>({ visible: false, onlyCopy: false })
```
[the remark this answers is not in the document]
Comment (daniel-hlavaty) [reply src/components/ticketList/TicketList.tsx:63]: @michalbella True, but for some reason TicketList is not using copyTicketButton. I didnt notice but you only added id into wallet.

File: src/components/ticketList/TicketListItem.tsx (hunk 1 above)
Comment (daniel-hlavaty) [reply src/components/ticketList/TicketListItem.tsx:167]: @michalbella Forgot to uncomment, will fix

**Pull request merged**